templates/download/risc-v/microchip-curiosity.html

```diff
@@ -0,0 +1,41 @@
+<div tabindex="0"
+     role="tabpanel"
+     id="microchip-curiosity-tab"
+     aria-labelledby="microchip-curiosity"
+     aria-hidden="true">
+  <div class="row p-section--shallow u-no-padding--top">
+    <div class="col-3">
+      <h2 class="u-hide--small">Microchip PIC64GX1000 Curiosity Kit</h2>
+    </div>
+    <div class="col-6 risc-v-container u-vertically-center u-align--center">
+      {{ image (
+      url="https://assets.ubuntu.com/v1/e0c220e9-0da5bc43-image001_reworked_20240701.png",
+      alt="Microchip Curiosity",
+      width="452",
+      height="300",
+      hi_def=True,
+      loading="lazy",
+      attrs={"class": "risc-v-image"}) | safe
+      }}
+    </div>
+  </div>
+  <div class="row p-strip u-no-padding--top">
+    <hr class="col-9" />
+    <div class="col-3">
+      <h3 class="p-heading--5 u-no-padding--top">
+        Ubuntu Server
+        <br class="u-hide--small" />
+        preinstalled image
+      </h3>
+    </div>
+    <div class="col-6">
+      <p>
+        <a class="p-button--positive"
+           href="https://people.canonical.com/~platform/images/microchip/pic64gx1000_curiosity_kit/ubuntu-24.04-preinstalled-server-riscv64+pic64gx.xz">Download 24.04</a>
```

Comment:
The download link is giving a 404. Perhaps the copy doc link is outdated, I'll leave a message on the docs to see if there's a new link for this.